serverlessworkflow/modules/ROOT/pages/cloud/enhanced-service-discovery.adoc

```diff
@@ -0,0 +1,136 @@
+= Kubernetes Service Discovery for Workflow Application Configuration
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, enhanced-service-discovery
+// links
+:quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
+
+
+The Kubernetes Service Discovery allows you to have a static URI that will define the Kubernetes resource that you use
+to perform HTTP requests, the following sketch demonstrates how this URI is composed:
```

Comment:
In this first paragraph, we should make it crystal clear that the service discovery feature kicks for all URIs that have the supported scheme in the Quarkus configuration properties values.